### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by Lino09**
## Bookstore: Project Setup

In this PR I add the following:

- Setup React APP
- Setup Redux
- Setup React Router
- Setup Linters
- Setup Working file tree according to Duck pattern

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  </div>
+);
+
+Book.propTypes = {
+  title: PropTypes.string.isRequired,
+  author: PropTypes.string.isRequired,
+  category: PropTypes.string.isRequired,
+  progress: PropTypes.string.isRequired,
+  currentChapter: PropTypes.string.isRequired,
+};
```
Comment (ArthurGC) [inline src/components/Book.js]: - [x] It would be better to fix the console error about propTypes. 🛠️ 

error-1

File: src/components/Book.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+const Book = ({
+  title, author, category, progress, currentChapter,
+}) => (
+  <div className="book">
+    <div className="book-col">
+      <small>{category}</small>
+      <h3 className="book-title">{title}</h3>
+      <smal className="book-author">{author}</smal>
+      <div className="book-opt">
```
Comment (ArthurGC) [inline src/components/Book.js]: - [x] Here, there is another console error too. 🛠️ 

error-2

**Review by ArthurGC — CHANGES_REQUESTED** [review]
# Required Changes ♻️ :warning: 
Hi  @Lino09 

Amazing job!
There are some issues that you still need to work on to go to the next project but you are almost there! 



## Good Points :+1: :
You used proper GitHub Flow :heavy_check_mark:
Your linter checks are all passing :heavy_check_mark:
Your PR messages are descriptive :heavy_check_mark:
Your commit messages are meaningful :heavy_check_mark:

## Aspects to improve ♻️
_Check the comments under the review._
- [x] Please, It is mandatory to follow the `ducks pattern` for creating the structure of the files.

ducks pattern

- The book and categories folders/files are missing. It's a good idea to create them too.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Lino09** [comment]
I dunno what I was thinking when submitting without checking the console, Thanks a lot Arthur!

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @Lino09,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Lino09** [comment]
@TedLivist I'm so sorry I was talking with my wife and got so distracted I pasted the URL in the feedback!
Thanks a lot for your help with my project!

**Comment by TedLivist** [comment]
Hey, @Lino09 no problem. It's more important that you progress.

**Pull request merged**